site/en/external/extension.md

```diff
@@ -174,6 +174,13 @@ several repo visibility rules:
         the repo visible to the module instead of the extension-generated repo
         of the same name.
 
+### Overriding and adding repos
+
+The root module can use
+[`override_repo`](/rules/lib/globals/module#override_repo) to override or add
```

Comment:
curious why you mentioned "adding repos" here. What does it mean for a repo to be added to an extension? I can't think of a case where it meaningfully matters.

If you do have a use case in mind, it would be nice to add it to the doc here (would be nice to have an example for overriding too, actually)

Reply:
I added examples and also split `override_repo` into `override_repo` and `inject_repo`.

Reply:
these are great, thanks!